main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Since we know that we will have 10 rounds of gameplay, consider using a for loop here. It may also be helpful to use a more explicit variable name than "loop".

If you want to keep a while loop, the conditional loop will need to be `while(loop < 11)` to ensure the game repeats 10 times.